packages/backend/src/api/v1/docs-content.controller.ts

```
@@ -71,12 +79,16 @@ export class ApiV1DocsContentController {
-  ): Promise<'doc' | 'slides'> {
+  ): Promise<'doc' | 'slides' | 'note'> {
     const meta = await this.documentService.getDocumentOrThrow({
       id: documentId,
       workspaceId,
     });
-    if (meta.type !== 'doc' && meta.type !== 'slides') {
+    if (
+      meta.type !== 'doc' &&
+      meta.type !== 'slides' &&
+      meta.type !== 'note'
+    ) {
```

ESLint

[error] Replace `⏎······meta.type·!==·'doc'·&&⏎······meta.type·!==·'slides'·&&⏎······meta.type·!==·'note'⏎····` with `meta.type·!==·'doc'·&&·meta.type·!==·'slides'·&&·meta.type·!==·'note'`

(prettier/prettier)